w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml

```diff
@@ -75,7 +75,7 @@ find_markers:
         reference: rest
         use_raw: 'true'
 find_markers_cell_type:
-    groupby: inferred_cell_type
+    groupby: CELL_TYPE_FIELD
```

Comment:
I'm guessing that if the scanpy module for finding marker genes doesn't find this field for groupby doesn't fail or the failure is handled at the level of data set collection filtering? Just wondering.

Reply:
The Galaxy workflow step that does cell type marker detection is allowed to fail.